salt/modules/virt.py

```diff
@@ -2205,6 +2232,32 @@ def init(
     min_guarantee
         the guaranteed minimum memory allocation for the guest
 
+    hugepages
+         memory allocated using ``hugepages`` instead of the normal native page size. The value of this parameter is the
+         size of the memory page. For example ``"hugepages": "1g"``.
```

Comment:
[libvirt doc](https://libvirt.org/formatdomain.html#memory-backing) mentions multiple huge pages pages and has them assigned to a cpuset. How comes this is not handled here?

Reply:
new code is updated to handle assigning of ``cpuset`` :)